src/processing/ParameterizedOperationsUtil.java

```diff
@@ -54,7 +54,7 @@ public static BoardObject colorChangeUtil(BoardObject obj, UserId id, Intensity
         BoardObjectOperation newBoardOp = new ColorChangeOperation(newIntensity);
 
         // remove previous object from maps.
-        BoardObject dummyObj = SelectDeleteUtil.removeObjectFromMaps(obj.objectId);
+        BoardObject dummyObj = BoardState.removeObjectFromMaps(obj.objectId);
```

Comment:
`BoardState`is not a static class like `SelectDeleteUtil` was supposed to be. You can get the object of class `BoardState` from `ClientBoardState` class. Call the `removeObjectFromMaps()`method on this object.

Reply:
CBS used.